src/dbnode/storage/database.go

```diff
@@ -708,8 +716,31 @@ func (d *db) QueryIDs(
 	query index.Query,
 	opts index.QueryOptions,
 ) (index.QueryResult, error) {
+	var (
+		sp    opentracing.Span
+		spCtx stdctx.Context
+	)
+
+	goCtx, exists := ctx.GoContext()
+	if exists {
+		sp, spCtx = opentracingutil.StartSpanFromContext(goCtx, dbQuerySpanID)
+		sp.LogFields(
+			opentracinglog.String("query", query.String()),
```

Comment:
Do we need to add the query to the span at every level? Same for `StartInclusive` and `EndExclusive`.

maybe I don't know how tracing works but I thought you could just add that information in one span and it was would associated with the entire trace and you wouldn't need to do it at each layer

Reply:
I believe logs are associated with just a single span, but I'll double check